test-resources/headless-core-stub/utils/src/services/client-configuration.ts

```diff
@@ -1,14 +1,15 @@
 import { getParametersValues } from "../parameter/get-parameters";
 import config from "./config";
-const { commonParameterPrefix, testResourcesParameterPrefix } = config;
+const { commonStackName } = config;
 
 export class ClientConfiguration {
     public static async getConfig(clientId: string) {
         const parameters = [
-            `/${commonParameterPrefix}/clients/${clientId}/jwtAuthentication/audience`,
-            `/${commonParameterPrefix}/clients/${clientId}/jwtAuthentication/issuer`,
-            `/${commonParameterPrefix}/clients/${clientId}/jwtAuthentication/redirectUri`,
-            `/${testResourcesParameterPrefix}/ipv-core-stub-aws-headless/privateSigningKey`,
+            `/${commonStackName}/clients/${clientId}/jwtAuthentication/audience`,
+            `/${commonStackName}/clients/${clientId}/jwtAuthentication/issuer`,
+            `/${commonStackName}/clients/${clientId}/jwtAuthentication/redirectUri`,
+            `/${commonStackName}/clients/${clientId}/jwtAuthentication/publicSigningJwkBase64`,
+            "/test-resources/ipv-core-stub-aws-headless/privateSigningKey",
```

Comment:
Why are we hardcoding the `testResourcesParameterPrefix` now?

Reply:
It's always the [same value](https://github.com/govuk-one-login/ipv-cri-pipeline-deployment/blob/main/config/shared/output.tf#L2) this make it easier to work with, when you have your own test resources stack

Reply:
Hmm I think in this context it would be very common to deploy you're own test resources stack and although it is the same value now we can't assume that will always be the case.

Is this being a param causing some sort of issue that we are trying to fix/